packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatBubbles.tsx

```diff
@@ -161,6 +161,13 @@ export const AssistantBubble: FC<{
                 rows: queryResults.rows,
                 type: queryExecutionHandle.data.type,
             }),
+            pivotColumns:
+                // TODO: fix this using schema
+                message.vizConfigOutput &&
+                'breakdownByDimension' in message.vizConfigOutput &&
+                typeof message.vizConfigOutput.breakdownByDimension === 'string'
+                    ? [message.vizConfigOutput.breakdownByDimension]
+                    : undefined,
```

Comment:
@joaoviana this and other places with similar comment need the zod schemas defined on back-end to be moved to common for typing. Will be done in a separate pr